_config/products.yml

```diff
@@ -462,11 +462,39 @@ jbt_core:
             #md5_url: .MD5
 
     luna:
+      4.2.3.Final:
+        update_site_url: http://download.jboss.org/jbosstools/updates/stable/luna/
+        marketplace_url: http://marketplace.eclipse.org/node/1617241
+        marketplace_install_url: http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1617241
+        #archived: true
+        release_date: 2015-04-01
+        supported_jbt_is_version: 4.2.0.Final
+        blog_announcement_url: /blog/TODO.html
+        documentation_url: http://tools.jboss.org/documentation/
+        zips:
+          - name: Update site (including sources) bundle of all JBoss Core Tools
+            file_size: 530MB
+            url: http://sourceforge.net/projects/jboss/files/JBossTools/jbosstools4.2.x/jbosstools-4.2.3.Final_2015-03-26_22-41-56-B370-updatesite-core.zip
+            md5_url: http://sourceforge.net/projects/jboss/files/JBossTools/jbosstools4.2.x/jbosstools-4.2.3.Final_2015-03-26_22-41-56-B370-updatesite-core.zip.MD5
+          - name: Sources zip of all JBoss Core Tools
+            file_size: 37.1MB
+            url: http://sourceforge.net/projects/jboss/files/JBossTools/jbosstools4.2.x/jbosstools-4.2.3.Final_2015-03-26_22-41-56-B370-src.zip
+            md5_url: http://sourceforge.net/projects/jboss/files/JBossTools/jbosstools4.2.x/jbosstools-4.2.3.Final_2015-03-26_22-41-56-B370-src.zip.MD5
+          - name: Update site (including sources) bundle of all Hibernate Tools
+            file_size: 105MB
+            url: http://sourceforge.net/projects/jboss/files/JBossTools/jbosstools4.2.x/jbosstools-4.2.3.Final_2015-03-26_23-05-30-B264-updatesite-hibernatetools.zip
+            md5_url: http://sourceforge.net/projects/jboss/files/JBossTools/jbosstools4.2.x/jbosstools-4.2.3.Final_2015-03-26_23-05-30-B264-updatesite-hibernatetools.zip.MD5
+          - name: Stand-alone BrowserSim
+            file_size: 15.8MB
+            url: http://sourceforge.net/projects/jboss/files/JBossTools/jbosstools4.2./jbosstools-4.2.3.Final-browsersim-standalone.jar
```

Comment:
Missing "x". Should be http://sourceforge.net/projects/jboss/files/JBossTools/jbosstools4.2.x/jbosstools-4.2.3.Final-browsersim-standalone.jar